components/brave_shields/browser/ad_block_component_installer.cc

```diff
@@ -199,6 +202,22 @@ void RegisterAdBlockDefaultResourceComponent(
       cus, base::BindOnce(&OnRegistered, kAdBlockResourceComponentId));
 }
 
+void CheckAdBlockComponentsUpdate() {
+  auto runner = base::SequencedTaskRunner::GetCurrentDefault();
+
+  runner->PostDelayedTask(FROM_HERE, base::BindOnce([]() {
+                            BraveOnDemandUpdater::GetInstance()->OnDemandUpdate(
+                                kAdBlockResourceComponentId);
+                          }),
+                          base::Seconds(base::RandInt(0, 10)));
```

Comment:
@simonhong 
I wonder why we don't synchronize all the updates.
AFAIK, the component updater is designed to process multiple updates in a bunch.

Reply:
@atuchin-m I didn't know we coud check update multiple components at once.
Could you point the example if possible? I'll try to find how to do it also.

Reply:
@simonhong see SequentialUpdateChecker in our code base.

Reply:
> @simonhong see SequentialUpdateChecker in our code base.

Oh, Thanks. I'll check it out!